azavea/raster-foundry
# Add tile visibility to Project model & migrations

## Overview

Add a tileVisibility field to the Project model

### Checklist

~- [ ] Styleguide updated, if necessary~

## Testing Instructions

 * Run migrations
* Verify that existing projects default to being private
* Verify that new projects are created with the field set to private

Connects #1206
